pallets/gear/src/tests.rs

```diff
@@ -587,7 +587,7 @@ fn delayed_send_user_message_payment() {
 fn delayed_send_user_message_with_reservation() {
```

Comment:
should be solved within the PR
https://github.com/gear-tech/gear/blob/f087d0e3762281124c09fb23d804fb0a2d984048/pallets/gear/src/tests.rs#L678